DevelopGitHub/GitHubExtension/GitHubExtension.Templates/ExtensionMethods/IEnumerableExtension.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+
+namespace GitHubExtension.Templates.ExtensionMethods
+{
+    public static class IEnumerableExtension
+    {
+        public static bool IsEmpty(this IEnumerable source)
```

Comment:
For example, if I send a request http://localhost:50859/api/templates/categories/4(from postman), but I have only 3 categories in DB,  I will receive NotFound().